README.md

```diff
@@ -99,18 +99,18 @@ The observation of the robots include internal states, target pose or camera ima
     <img src="media/icub_walk.gif" width="320" height="240" alt="pioneer"/>
 
 ### Training Examples with [Agent Learning Framework (Alf)](https://github.com/HorizonRobotics/alf)
-Train simple navigation task with [Alf actor-critic](examples/ac_simple_navigation.gin)
+Train goal_task task with [Alf actor-critic](examples/ac_simple_navigation.gin)
```

Comment:
This results in an invalid link.  Maybe use absolute path?  https://github.com/HorizonRobotics/alf/blob/master/alf/examples/ac_simple_navigation.gin

Reply:
This is a local repo link, I've renamed ac_simple_navigation.gin and forgot to change the link here...

Reply:
Fixed now.